### DISCUSSION TITLE
Firefox 102, Fedora 35, Sbcl 2.2.6 is clog builder supposed to look like this?

### DISCUSSION THREAD

**Discussion by dtenny** [rabbibotton/clog#202] [Q&A] [answered]
![clog-builder-Screenshot_20220806_141948](https://user-images.githubusercontent.com/6500367/183261593-77b87a6f-07bf-4695-b6b2-2b714f8c4dca.png)
The clog builder I experience on Firefox doesn't look or behave quite like some of the tutorial docs describe, so I cloned the clog, clog-ace, and clog-terminal repos and and linked them into my quicklisp local projects, `(clog:clog-install-dir)` looks right after loading, but the builder still look wrong.  For example, the panels on the left have no scroll bar you can interact with, and no apparent title bar. [Update, I now see that for them to appear I have to have the mouse over the panel, and then further mouse over the small scroll bar to make it a fully visible with the ... swim lane ... for lack of a better word on my part].

Are there known problems with firefox? I don't have any notable extensions, and turned off my 'dark reader' extension to be sure. Anyway, an image of the clog builder screen is attached.  I'm new here, hopefully this was the best place to ask.

Firefox 102, Fedora 35, Sbcl 2.2.6

**Comment by dtenny** [comment]
Tried an old Google Chrome I had, looks mostly the same with clog builder, but the scroll bars are evident and fully visible with swim lane even if the mouse isn't over the panel.

Since upgrading to the recent clog git repos (vs the July 7 quicklisp version), my divs don't have "content" properties any more, just "text", and if I put `<h1>my text</h1>` in the text property, it eventually renders as an h1 but without the HTML markup in the property list (but not on the web page, the h1 isn't observed).  See attached image.
![text-property-Screenshot_20220806_181546](https://user-images.githubusercontent.com/6500367/183267643-c46428fe-0cc8-4bb9-ac7c-5d383ed51fac.png)

**Comment by rabbibotton** [comment, marked as answer]
I was just able to try for a sec. Everything seems to work perfect except for one visual anomaly same as you have in the top left corner the right side should be a black bar but the list box is off a bit.

The version of the builder has had much work since the tutorials. However all should still work as it does in tutorials. The project window lets you load directly your files etc and run the resulting app, but can close if desire.

**Comment by rabbibotton** [comment]
I will address the slight anomaly in the control palette and more thoroughly test tomorrow or Monday. Let me know if having any more issues on FireFox or in general.

**Reply by dtenny** [reply]
And the 'text' vs 'content' property on the divs? The content property seems to have disappeared (where it used to be at the bottom).  Thanks for looking.

**Reply by dtenny** [reply]
Also the `<h1>stuff</h1>` that doesn't render as headers in the div, only, ironically, in the 'text' property box after a page refresh.

**Reply by rabbibotton** [reply]
I am redoing the properties overall soon to improve them, add drop downs, etc. the "content" property was removed and only text offered (the actually text "node" the value that applies to that control. If someone wanted to use straight html they should use the Custom HTML block, and there your h1 element will render properly and you can edit it etc. I will be adding a full text editor for the html block soon.

**Reply by rabbibotton** [reply]
<img width="498" alt="image" src="https://user-images.githubusercontent.com/11616789/183293855-5d2b3925-c801-40ea-8de6-eddfe24ed881.png">
<img width="425" alt="image" src="https://user-images.githubusercontent.com/11616789/183293879-fd5b4a7e-6ef8-46a5-874e-03cc97a4ac98.png">

**Reply by dtenny** [reply]
Thanks.  I was working through the web page tutorial where it suggested putting some H1 text in the content property and it showed that reflected in the web page, and I wasn't getting that so was wondering. Parroting back what I'm hearing, the attribute is now 'text', but for an H1 to work I need to use a custom HTML block.  All set.  I'm impressed by the amount of energy you've put into this, I'd been eyeballing your reddit posts for a long while but this is the first time I've tried it.

**Reply by rabbibotton** [reply]
I think once I am done with properties going to need to redo or create better tutorials.

**Comment by rabbibotton** [comment]
Yes.

CLOG itself is stable. For the most part the Builder is as well in the
sense of basics but this particular round of work will affect abit of
difference in the properties towards my goal if simplicity :)

**Comment by rabbibotton** [comment]
I have made additional fixes today for rendering issues in firefox and safari, if anything still persists please let me know.
